controllers/teams/deleter.js

```diff
@@ -1,27 +1,41 @@
-const { Team } = require('../../databases/db');
+const { team} = require('../../databases/db');
 const { imgController, fileController } = require('../filesControllers'); 
 const path = require('path');
-
+const responses= require('../../utils/responses/responses');
+const Sticker= require('../../databases/db');
+const game= require('../../databases/db');
 /**
  * If the team exists, delete it and send a 200 response, otherwise send a 404 response.
  * @param req - The request object.
  * @param res - the response object
  */
 const destroy = async (req, res) => {
   const teamId = req.params.teamId;
-  const team = await Team.findByPk(teamId);
-  if (team) {
-    const { badge: fileurl } = team;
+  const Team = await team.findByPk(teamId);
+
+  const sticker = await Sticker.findOne({
+    raw:true,
+    where: {teamId : req.params.teamId}
+  });
+  const Game = await game.findOne({
+    raw:true,
+    where: {teamId : req.params.teamid}
+  });    
+  if (sticker || Game) {
+      return responses.errorDTOResponse(res, 400, "No se puede eliminar el equipo porque tiene jugadores o partidos asociados");
+  }
```

Comment:
```suggestion
  const team = await Team.findByPk(teamId);
  
  if (typeof team === "undefined" || team == null)
    return responses.errorDTOResponse(res, 404, "Equipo no encontrado");
  
  const sticker = await team.getSticker();
  const game = await team.getGame();
      
  if (sticker || game)
    return responses.errorDTOResponse(res, 400, "No se puede eliminar el equipo porque tiene jugadores o partidos asociados");
```
Considera esta solucion